test/run-tests

```diff
@@ -106,9 +106,20 @@ def handle_transient_httperrors(error):
         # If this is error from GitHub, there should be some explanation in response
         # content. Try to print it.
         try:
-            print("E: GitHub: " + error.response.json()["message"])
+            github_msg = error.response.json()["message"]
         except Exception:
-            pass
+            github_msg = ""
+
+        if "abuse detection" in github_msg:
+            timeout = transient_error_wait_time.next()
+            print(
+                ">>> We have triggered abuse detection, waiting {} s...".format(timeout)
+            )
+            time.sleep(timeout)
+            return True
+
+        if github_msg:
```

Comment:
How about always printing the error message, i.e. moving this code before the abuse detection handling? Having the raw error message might be useful for debugging.

Reply:
Done